compiler/test/compilable/cppmangle.d

```diff
@@ -666,6 +666,8 @@ extern (C++) class C18890_2
     Agg s;
 }
 
+void test18890()
+{
```

Comment:
what is with the changes to this file? they seem unrelated.

Reply:
It's an ordering issue on when the inference is done. In this case, the test was done before the inference on the functions. So I put them inside a function, where the static checks are still done, but the inference happens earlier.

Reply:
OK that's fine, but please to add an appropriate changelog entry that explains this if it might break stuff.